versioned/storage/bigtable/src/main/resources/org/projectnessie/versioned/storage/bigtable/Dockerfile-google-cloud-sdk-version

```diff
@@ -1,3 +1,5 @@
 # Dockerfile to provide the image name and tag to a test.
 # Version is managed by Renovate - do not edit.
-FROM docker.io/google/cloud-sdk:463.0.0
+# See https://cloud.google.com/sdk/docs/downloads-docker#docker_image_options
+# Use debian_component_based because it supports linux/arm
+FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:463.0.0-debian_component_based
```

Comment:
Historically, there has been is a slight permeation of test classes within production ones in storage modules, cf. implementations of `BackendTestFactory`: they are all declared in `src/main`, I think for simplicity sake. I can look into that, but that would require another PR since all backends are concerned.

Reply:
I think it would be nice to move those dockerfiles to test sources... I assume they get included into published main jars ATM.

Reply:
Looking into it now FYI.